plugins/3MFReader/ThreeMFWorkspaceReader.py

```diff
@@ -616,6 +616,7 @@ def preRead(self, file_name, show_dialog=True, *args, **kwargs):
             # The machine included in the project file exists locally already, no need to change selected printers.
             is_networked_machine = global_stack.hasNetworkedConnection()
             is_abstract_machine = parseBool(existing_global_stack.getMetaDataEntry("is_abstract_machine", False))
+            self._dialog.setMachineToOverride(global_stack.getId())
```

Comment:
Do we still need this? I thought the fix from https://github.com/Ultimaker/Cura/pull/13848/commits/4d2f6b9655dd4cecf9eef0f1f808b05e3b500d28#diff-bc57941f6914b8706cf435c14b5d522814678acc5f510b69653cf5a538e9839aR735 already fixed this?